src/components/icons/logo-white/logo-white.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export const LogoWhite = () => (
```

Comment:
Why change these svgs into components?

Reply:
I was trying to avoid the pixelation of the logos, I put components because I thought Gatsby might put some wrong value to the img. So, Do I change it for an img again?